testing/ostest/signest.c

```diff
@@ -52,6 +53,8 @@
  * Private Data
  ****************************************************************************/
 
+static pid_t waiterpid;
```

Comment:
should we change to pthread_t

Reply:
yes, I believe it is more appropriate. I was just  copying the old code...